lib/galaxy_test/api/test_drs.py

```diff
@@ -9,6 +9,8 @@
     urlparse,
 )
 
+import requests
```

Comment:
I think we should add the header here.

Reply:
I think it's only used to connect to Galaxy's `/api/ga4gh/drs/v1/*` and `/api/drs_download/{object_id}` URLs.